keys.md

```diff
@@ -0,0 +1,37 @@
+Authentication for the Termly API uses 2 keys - a public and private. The public key is sent with each request. The private key is used to generate [signatures](signature.md) that Termly uses to verify the request.
+
+Each public key will have AT MOST one unexpired private key. If your key is compromised, you can immediately expire it and get a new key. If you are doing regularly scheduled key rolling, you can create a new private key for the public key. The old private key will be given an expiration 30 days from the date the new key is created. This expiration can be changed to be shorter or longer. Thirty days is just the default.
```

Comment:
Confused, the tables in the internal documentation show that multiple keys can be active simultaneously...

Reply:
"Unexpired" means there is an expiration date, not necessarily that it is expired...?

Reply:
In my head I was thinking unexpired meant missing.